### ISSUE TITLE
Incorrect poses for Courthouse scene?

### ISSUE THREAD

**Issue by hanyucc** [isl-org/TanksAndTemples#55]
Hi, I was visualizing the camera poses for the courthouse scene after converting the .log file into Colmap format. However, I noticed that some of the camera poses don't look right. For example, these are two images that are supposed to be from roughly the same viewpoint (the image planes highlighted in pink), but that doesn't seem to be the case.

The flag pole in the first image seems to suggest that the photo is taken from the left side of the scene, but the visualized camera pose is on the right side instead. (In fact there are no camera poses in the left side of the scene, which seems to be problematic)

![image](https://github.com/isl-org/TanksAndTemples/assets/22674006/63c37a83-b30c-4ed8-97e2-103e2c7d67f9)
![image](https://github.com/isl-org/TanksAndTemples/assets/22674006/b95dddbe-eadb-49d8-b9c2-240b24667c83)

I'm wondering if you could check if this is actually an issue with the dataset, or if I'm doing something incorrectly during the conversion. Thanks!

**Comment by akshay-krishnan** [comment]
@hanyucc I noticed the same! It seems like some cameras poses are completely off, on the wrong side of the building.

**Comment by arknapit** [comment]
Yes, this scene has a few wrong camera poses. These poses are just extracted from a standard Colmap run. For our purpose of pre-aligning the dense pointclouds it is usually sufficient, but you should run your own SfM and verify it, e.g. if you do an image/camera position based evaluation.
